i18n-xgettext-asm/src/main/java/net/jhorstmann/i18n/xgettext/asm/ConstantTrackingInterpreter.java

```diff
@@ -373,6 +378,11 @@ public Value naryOperation(AbstractInsnNode insn, List values) throws AnalyzerEx
         if (opcode == MULTIANEWARRAY) {
             return REFERENCE_VALUE;
         } else {
+//            TODO
+//            if( insn instanceof InvokeDynamicInsnNode) {
+//                InvokeDynamicInsnNode methodInsn = (InvokeDynamicInsnNode) insn;
```

Comment:
You can probably just return `REFERENCE_VALUE` here, we won't be handing the result of an invokedynamic in a special way.